tools/modules/system/manage_dtoverlays.sh

```
@@ -18,12 +18,14 @@ function manage_dtoverlays () {
 	# check if user agree to enter this area
 	local changes="false"
 	local overlayconf="/boot/armbianEnv.txt"
-	# Raspberry Pi has different name
-	local overlaydir
-	overlaydir=$(find /boot/dtb/ -maxdepth 1 -type d \
-	\( -name "overlay" -o -name "overlays" \) | head -n1)
-	local overlay_prefix
-	overlay_prefix=$(awk -F= '/^overlay_prefix=/ {print $2}' "$overlayconf")
+	if [[ "${LINUXFAMILY}" == "bcm2711" ]]; then
+		# Raspberry Pi has different name
+		local overlaydir=$(find /boot/dtb/ -maxdepth 1 -type d \( -name "overlay" -o -name "overlays" \) | head -n1)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)